src/lib/sns.js

```diff
@@ -25,8 +25,61 @@ const attrsFromPayload = function (payload) {
     collection: {
       DataType: 'String',
       StringValue: collection
+    },
+  }
+
+  const bbox = getBBox(payload.record)
+  if (bbox) {
+    attributes['bbox.ll_lon'] = {
```

Comment:
could you change `ll` to `sw` and `ur` to `ne`? It's sort of a pedantic point, but I prefer to use cardinal directions instead of left/right and lower/upper, as a GeoJSON bbox is defined this way.

Reply:
let's change this to sw and ne

Reply:
👍 I switched `ll_` to `sw_` and `ur_` to `ne_` in fixup 2abc1f9